### ISSUE TITLE
Unable to express anger in ASCII without using a backslash escape, interpreted as quote by Markdown.

### ISSUE THREAD

**Issue by Cloudwalk9** [vector-im/element-web#20724]
#### What would you like to do?
Express anger to my friends using ASCII emotes, such as `>:( >:[ >:c` and others

#### Why would you like to do it?
Because I'm mad.

#### How would you like to achieve it?
Requiring a space between the '>' and the quoted text. Discord does this fwiw

### Additional context

>:(

**Comment by t3chguy** [comment]
> Requiring a space between the '>' and the quoted text. Discord does this fwiw

We use an external markdown specification called CommonMark so that you don't have to learn a new dialect for each product you use

https://spec.commonmark.org/dingus/

There is another open issue about switching to a different dialect but it suggests GFM which has the same behaviour.

Running a bespoke/modified dialect would be a lot of extra work for very little gain and with a lot of added user-facing confusion.

**Comment by anoadragon453** [comment]
Personally, I've seen different friends hit this bug many a time - they're used to other chat apps which don't create a block quote when typing `>x`, like Telegram and Discord. Telegram doesn't support block quotes, Discord will only create one if a space is added afterwards. 

But due to Discord's live markdown preview, this surprise is experienced before a message is sent.

https://user-images.githubusercontent.com/1342360/150884738-056f2da0-1d7b-4c97-a6d2-1753d1567649.mp4

which saves one from finding out *after* having sent a message. (this is tracked at https://github.com/vector-im/element-web/issues/20360).

The same argument can be made for <del>strikethrough</del>. Users expect `~~this syntax~~` to work. We're also still lacking a way to send inline spoilers (https://github.com/vector-im/element-web/issues/10882).

I disagree that a custom dialogue would be little gain, as it would allow us to support the above syntax. I'm also unconvinced that it would be too much work to support. Other chat apps do it -- why can't Element?


[markdown-it](https://github.com/markdown-it/markdown-it/) is a markdown parser with support for plugins for custom syntax. [discord-markdown](https://www.npmjs.com/package/discord-markdown) is a from-scratch library that tries to mimic discord's markdown syntax.

**Comment by t3chguy** [comment]
The issue ends up being parity matching between all the platforms, right now we already have 3 different dialects but in theory just using stock commonmark makes it easier to match between them. Having to mimic the extensions for both serialising and deserializing across all 3 platforms adds a lot of load to maintain.

**Comment by anoadragon453** [comment]
Parity in syntax between platforms is indeed something that has been a problem in the past. Though, did the difficultly in implementation lie in the conversion between text and HTML, or in implementing the rendering? If the latter, then that's a separate problem from commonmark vs. GFM etc. 

If it is down to the markdown renderer, then much like the spread of matrix-rust-sdk, could we not maintain a single rust library which takes in text and spits out HTML (and vice versa) and use that across all three platforms?

**Comment by t3chguy** [comment]
HTML Rendering implementation is an entirely different topic, the Matrix spec has a set of suggested tags to support and those are all supported in Web, with partial support elsewhere. The main difficulty was a stable MD -> HTML -> MD solution which was consistent between the platforms. Just checked, Android has finally moved to stock CommonMark too now, from their MarkedJS-in-Android hell from [yesteryear](https://github.com/vector-im/riot-android/blob/develop/vector/src/main/java/im/vector/util/VectorMarkdownParser.java)

> could we not maintain a single rust library which takes in text and spits out HTML

Yes, probably, but then also need to maintain our own documentation which we are awful at. We currently just point people at the stock CommonMark docs.

**Comment by ShadowJonathan** [comment]
#5940 is relevant here, as commonmark enforces this exact behaviour.

I'm wholly in support for the new syntactical approach btw, if i can help with that, please let me know :+1:

**Comment by ashkitten** [comment]
> You can use `/plain` or disable MD in Preferences

i don't really consider this a good option. i run up against this inconvenience almost daily and it's always a bit of an "ugh element" moment with me and my friends. given that no other chat app (discord, slack, zulip, ...) has this issue, i find it pretty frustrating that the official answer is that it's intended behavior...

**Comment by anoadragon453** [comment]
Indeed. Plus it means the rest of your message won't be interpreted as markdown, which is often undesirable.

I think a reasonable alternative would be requiring require a space after the `>` to intend a quote, e.g. `> quote` vs. `>_>`.

**Comment by t3chguy** [comment]
> I think a reasonable alternative would be requiring require a space after the > to intend a quote, e.g. > quote vs. >_>.

We won't be maintaining our own markdown dialect, I suggest adding comments to https://github.com/vector-im/element-meta/issues/330 that you would like to pick a markdown dialect which only treats blockquotes as `> ` with a determinate space after the gt
